cfme/tests/webui/test_general_ui.py

```diff
@@ -509,6 +529,53 @@ def test_provider_details_page_refresh_after_clear_cookies(
     assert login_view.is_displayed
 
 
+@pytest.mark.tier(1)
+@pytest.mark.provider([InfraProvider], selector=ONE_PER_CATEGORY)
+@pytest.mark.parametrize("option", ALL_OPTIONS)
+def test_infrastructure_provider_left_panel_titles(
+    setup_provider, provider, option, soft_assert
+):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/18h
+        testSteps:
+            1. Add an infrastructure provider and navigate to it's Details page.
+            2. Select Properties on the panel and check all items, whether they have their titles.
+            3. Select Relationships on the panel and check all items,
+                whether they have their titles.
+        expectedResults:
+            1.
+            2. Properties panel must have all items and clicking on each item should display
+                the correct page.
+            3. Relationships panel must have all items and clicking on each item should display
+                the correct page.
+    """
+    view = navigate_to(provider, "Details")
+    accordion = getattr(view.entities.sidebar, option)
+
+    for panel in ALL_OPTIONS[option]:
+        # Sometimes the panel doesn't enable immediately,
+        # using wait_for to give it some time to enable
+        wait_for(
+            lambda: not accordion.tree.is_disabled(partial_match(panel)),
+            fail_func=view.browser.refresh,
+            num=40,
+            delay=2,
+            message="Wait until the panel enables",
+            handle_exception=True,
+            silent_failure=True
+        )
+        if not accordion.tree.is_disabled(partial_match(panel)):
+            accordion.tree.select(partial_match(panel))
+            test_view = provider.create_view(ALL_OPTIONS[option][panel], wait="60s")
```

Comment:
Since you passed `wait` parameter, this will raise an exception before your soft assert if the view isn't displayed.